test/pytest/test_xpedite/test_profiler/scenario.py

```diff
@@ -252,7 +251,7 @@ def __init__(self):
     self._scenarios = OrderedDict()
     self.remote = None
 
-  def loadScenarios(self, runPath, apps, scenarioTypes, remote=None):
+  def loadScenarios(self, runPath, apps, scenarioTypes=None, remote=None):
```

Comment:
why default scenarioTypes to None ? would this method fail to load any scenarios ?